packages/packages/recently-edited/src/components/top-bar/chip-doc-type.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+	ArchiveTemplateIcon,
+	HeaderTemplateIcon,
+	FooterTemplateIcon,
+	PostTypeIcon,
+	PageTypeIcon,
+	PopupTemplateIcon,
+	SearchResultsTemplateIcon,
+	Error404TemplateIcon,
+	LoopItemTemplateIcon,
+	LandingPageTemplateIcon,
+	PageTemplateIcon,
+	SectionTemplateIcon,
+	ContainerTemplateIcon,
+} from '@elementor/icons';
+import { Chip } from '@elementor/ui';
+
+const iconsDocType = {
+	header: HeaderTemplateIcon,
+	footer: FooterTemplateIcon,
+	single_post: PostTypeIcon,
+	single_page: PageTypeIcon,
+	popup: PopupTemplateIcon,
+	archive: ArchiveTemplateIcon,
+	search_results: SearchResultsTemplateIcon,
+	loop_item: LoopItemTemplateIcon,
```

Comment:
header, footer, popup, loop - we need to verify with @KingYes that it's ok to include "Pro" content in the "Core".